internal/show/brokers.go

```diff
@@ -70,8 +62,8 @@ func Brokers(clusterInfo *cluster.Info, _ string, status reporter.Interface) err
 		)
 	}
 
-	status.End()
 	printer.Print()
+	status.End()
```

Comment:
Why was this moved? Actually either way this call is unnecessary b/c there's no status started at this point so we can remove it.